app/scss/style.scss

```diff
@@ -64,7 +67,29 @@
   }
 }
 
+@media screen and (min-width: 1080px) {
```

Comment:
Do the mixin, and only inside the selector for that item. See the other existing ones for examples.

Reply:
Done